### DISCUSSION TITLE
Convert package-lock.json or yarn.lock to deno.lock

### DISCUSSION THREAD

**Discussion by alexgleason** [denoland/deno#25814] [Q&A]
I have a node project with a lot of dependencies that I will finally move to Deno for 2.0.

But Deno creates a new deno.lock from my package.json, essentially upgrading 180 packages at once.

I have a yarn.lock. I can convert between yarn.lock and package-lock.json. Can I convert either to deno.lock?

**Comment by bartlomieju** [comment]
I opened https://github.com/denoland/deno/issues/25815.
